### PR TITLE
DROOLS-733 - Added test verifying task comments.

### PR DESCRIPTION
**Pull request by sutaakar**
Shows marshalling error in JAXB marshaller - cannot marshall TaskComment.

### CODE REVIEW

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/UserTaskServiceIntegrationTest.java [hunk 1]
```diff
@@ -684,6 +685,60 @@ public void testUserTaskSetTaskProperties() throws Exception {
...
+     */
+    @Test
+    public void testUserTaskComments() throws Exception {
+        assertSuccess(client.createContainer("definition-project", new KieContainerResource("definition-project", releaseId)));
+
+        Long processInstanceId = processClient.startProcess("definition-project", "definition-project.usertask");
+        assertNotNull(processInstanceId);
+        assertTrue(processInstanceId.longValue() > 0);
+        try {
+            List<TaskSummary> taskList = taskClient.findTasksAssignedAsPotentialOwner("yoda", 0, 10);
```
Comment (triceo) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/UserTaskServiceIntegrationTest.java]: @sutaakar, Can you make "yoda", "First comment" etc. into constants? Will make the test easier to understand and maintain.

Comment (sutaakar) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/UserTaskServiceIntegrationTest.java]: @triceo Can do, some things can be turned into test suite wide constants, others just local ones.

**Comment by sutaakar** [comment]
Refactored according to comments, added deleteTaskComment() verification.

**Comment by mswiderski** [comment]
merged and fixed the marshaling issues. thanks @sutaakar

**Pull request closed without merging**